recipe/meta.yaml

```diff
@@ -41,6 +42,7 @@ requirements:
     - filelock
     - pytorch =*=cuda*
     - lit
+    - cudatoolkit
```

Comment:
Yeah, this needs to be broken down, but for that we'll need some input regarding the new CUDA setup.